homework_postgres/PostgreSQL-Students.ipynb

```diff
@@ -40,9 +40,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.                                    INNER JOIN roles ON person.id=roles.personId
</pre><p>The same question above, does inner join safe here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/haicheviet/dataeng-course-1/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='142'/>